apps/front/src/templates/signup-form/logic/customErrorMap.ts

```diff
@@ -15,17 +15,23 @@ export const customErrorMap = (
   issue: ZodIssueOptionalMessage,
   ctx: ErrorMapCtx
 ): ErrorMapResult => {
-  if (issue.message && translationsKeys.includes(issue.message)) {
+  if (issue.message && formsKeys.includes(issue.message)) {
     return {
       message: issue.message,
     };
```

Comment:
thats what we're now doing in hublo-front, but here, you'll have an untranslated key right ?
I've tested and re-tested error message handling with zod, you can have a look at the final version in `useFormResolver.ts`